Packages/manifest.json

```diff
@@ -5,6 +5,7 @@
     "com.unity.ide.rider": "1.2.1",
     "com.unity.ide.visualstudio": "2.0.3",
     "com.unity.ide.vscode": "1.2.2",
+    "com.unity.quicksearch": "1.5.4",
```

Comment:
What's this package for?

Reply:
Searching Unity for things. It's basically Ctrl-Shift-F for looking for menu items, etc.

Reply:
Ok